src/styles/index.scss

```diff
@@ -1,3 +1,20 @@
+@import './blocks/variables';
+@import './blocks/color';
+@import './blocks/calendar';
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
Try don't use * selector (it impacts performance) better use ``html`` or ``body`` tag.

![image](https://github.com/mate-academy/layout_calendar/assets/54860641/bd5e9869-8b9c-4a6a-9c76-4f42f673137e)